src/monster.php

```diff
@@ -665,7 +665,8 @@
 			if ($morreu != 5 && $matou != 5 && $otroatak != 5 && $bixo->vez == 'e') {
 				include(__DIR__ . "/battle/levahit.php");
 				include(__DIR__ . "/battle/menosturno.php");
-				$db->execute("update `bixos` set `vez`='p' where `player_id`=?", [$player->id]);
+				// $db->execute("update `bixos` set `vez`='p' where `player_id`=?", [$player->id]);
+				$db->execute("update `bixos` set `vez`='p', `type`= 0 where `player_id`=?", array($player->id)); //Ajustado query, trocando o type para "0", onde cancela o loop de batalha sem poder usar as skills apos ataque.
```

Comment:
Let's use the comments in English so we can start distributing to global community